modules/videoio/src/cap_android_camera.cpp

```diff
@@ -437,6 +430,12 @@ class AndroidCameraCapture : public IVideoCapture
                     return submitRequest(ACaptureRequest_setEntry_i32, ACAMERA_SENSOR_SENSITIVITY, sensitivity);
                 }
                 return false;
+            case CAP_PROP_ZOOM:
```

Comment:
I propose to document the option for Android in https://github.com/opencv/opencv/blob/ab77e1cfc87d7908c09fa4d11cbeaac9e4c663fb/modules/videoio/include/opencv2/videoio.hpp#L171.